apps/cowswap-frontend/src/modules/tokensList/pure/FavouriteTokensList/index.tsx

```diff
@@ -18,8 +16,8 @@ export function FavouriteTokensList(props: FavouriteTokensListProps) {
   return (
     <div>
       <styledEl.Header>
-        <h4>Favorite tokens</h4>
-        <InfoIcon iconType="help" content="Your favorite saved tokens. Edit this list in your account page." />
+        <h4>Favorite tokens</h4>        
```

Comment:
Nitpick: extra space 🤓 

```suggestion
        <h4>Favorite tokens</h4>
```